### DISCUSSION TITLE
Set user Password after OAuth2 login

### DISCUSSION THREAD

**Discussion by onlyspike** [pocketbase/pocketbase#2862] [Q&A]
Hi,

How can I set a user password after a user has logged in  with OAuth3.
I dont want to go through the whole reset password email chain process.
Is this possible?

**Comment by ganigeorgiev** [comment]
There is no web api for that at the moment (unless you perform the change as admin or the user has Manage API rule access). Currently this is an existing limitation that will be eventually resolved sometime in the future as part of a future refactoring (see https://github.com/orgs/pocketbase/projects/2/views/1?pane=issue&itemId=22401932).
